app/src/main/java/com/kickstarter/libs/utils/NumberUtils.java

```diff
@@ -109,6 +109,10 @@ private static Boolean isWholeNumber(final @NonNull Double value) {
     return value == Math.round(value);
   }
 
+  /**
+   * @deprecated Use com.kickstarter.libs.utils.extensions.StringExt#parseToDouble(java.lang.String)
+   */
+  @Deprecated
```

Comment:
The functions this calls is deprecated, so this should be marked deprecated was my approach